tempodb/backend/tenantindex.go

```diff
@@ -64,3 +61,43 @@ func (b *TenantIndex) unmarshal(buffer []byte) error {
 	d := json.NewDecoder(gzipReader)
 	return d.Decode(b)
 }
+
+func (b *TenantIndex) marshalPb() ([]byte, error) {
+	buffer := &bytes.Buffer{}
+
+	z, err := zstd.NewWriter(buffer)
```

Comment:
Did we do any benchmarks on zstd vs other?  We resulted with snappy in parquet because it had the best balance of speed vs resources.  Not blocking, because we could change it later easily by switching on the file name, but would be interested to see benchmarks if we have them.

Reply:
Hmm, I didn't benchmark, but I can.  We could also push this into an override so we can swap out the implementation without a rollout, then add more implementations later if we want.